test/phoenix/live_dashboard/system_info_test.exs

```diff
@@ -66,6 +89,42 @@ defmodule Phoenix.LiveDashboard.SystemInfoTest do
     end
   end
 
+  describe "process filter" do
+    setup do
+      Application.put_env(:phoenix_live_dashboard, :process_filter, ProcessFilter)
+      on_exit(fn -> Application.put_env(:phoenix_live_dashboard, :process_filter, nil) end)
+    end
+
```

Comment:
In these tests, you're changing the application environment which is the same for all the tests. For this reason, these tests cannot be `async: true` because can affect other running tests. In this kind of situation, I create a new module in this same file:

```
defmodule Phoenix.LiveDashboard.SystemInfoSyncTest do
  use ExUnit.Case, async: false
```